playwrightModules-demo/pages/SauceAppPages/contactListPage.ts

```diff
@@ -0,0 +1,29 @@
+import { Page, Locator } from "playwright";
+
+export class ContactListPage {
+    readonly page: Page;
+    readonly firstNameInput: Locator;
+    readonly lastNameInput: Locator;
+    readonly dateOfBirth: Locator;
+    readonly emailInput: Locator;
+    readonly phoneInput: Locator;
+    readonly submitButton: Locator;
+
+    constructor(page: Page) {
+        this.firstNameInput = page.locator('[id="firstName"]');
+        this.lastNameInput = page.locator('[id="lastName"]');
+        this.dateOfBirth = page.locator('[id="birthdate"]');
+        this.emailInput = page.locator('[id="email"]');
+        this.phoneInput = page.locator('[id="phone"]');
+        this.submitButton = page.locator('[id="submit"]');
+    }
+
+    async addContact(person: any) {
```

Comment:
Use of any is discourage :). The more optimal way would be to define a "Person" interface or type that describes the structure of the person object.

interface Person {
    firstName: string;
    lastName: string;
    dateOfBirth: Date;
    email: string;
    phone: string;
}

async addContact(person: Person) {}